Makefile: bump test timeouts

otan opened the pull request:
Tests like logictestccl take 20-30 minutes, some of them pushing to the
limits of the 30 minute timeout. Extend the timeout to account for this.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/67634)
<!-- Reviewable:end -->

otan committed:
Makefile: bump test timeouts

Tests like logictestccl take 20-30 minutes, some of them pushing to the
limits of the 30 minute timeout. Extend the timeout to account for this.

Release note: None

otan commented:
thanks!

bors r=rail
